core/src/main/java/org/bitcoinj/core/Peer.java

```diff
@@ -569,53 +573,50 @@ protected void processMessage(Message m) throws Exception {
+        else /*if (m instanceof SporkMessage) {
+            sporkManager.processSpork(this, (SporkMessage)m);
+        } else if (m instanceof SyncStatusCount) {
+            masternodeSync.processSyncStatusCount(this, (SyncStatusCount)m);
+        } else if (m instanceof GovernanceSyncMessage) {
             //swallow for now
-        } else if(m instanceof GovernanceObject) {
+        } /*else if (m instanceof GovernanceObject) {
             context.governanceManager.processGovernanceObject(this, (GovernanceObject)m);
-        } else if(m instanceof GovernanceVote) {
+        } else if (m instanceof GovernanceVote) {
             context.governanceManager.processGovernanceObjectVote(this, (GovernanceVote)m);
         } else if (m instanceof SimplifiedMasternodeListDiff) {
             context.masternodeListManager.processMasternodeListDiff(this, (SimplifiedMasternodeListDiff) m);
-        } else if(m instanceof InstantSendLock) {
+        } else if (m instanceof InstantSendLock) {
             context.instantSendManager.processInstantSendLock(this, (InstantSendLock) m);
-        } else if(m instanceof ChainLockSignature) {
+        } else if (m instanceof ChainLockSignature) {
             context.chainLockHandler.processChainLockSignature(this, (ChainLockSignature)m);
-        } else if (m instanceof SendHeadersMessage) {
+        } else */ if (m instanceof SendHeadersMessage) {
             // We ignore this message, because we don't announce new blocks.
         } else if (m instanceof SendAddressMessageV2) {
             // We ignore this message, because we don't reply to sendaddrv2 message.
-        } else if (m instanceof QuorumRotationInfo) {
+        } else /*if (m instanceof QuorumRotationInfo) {
             context.masternodeListManager.processQuorumRotationInfo(this, (QuorumRotationInfo) m, false, null);
         } else if (CoinJoinManager.isCoinJoinMessage(m)) {
             context.coinJoinManager.processMessage(this, m);
-        } else {
+        } else*/ {
             log.warn("{}: Received unhandled message: {}", this, m);
         }
     }
 
-    protected void processInstantSendLock(InstantSendLock islock) {
-        if (currentFilteredBlock != null) {
-            if (!currentFilteredBlock.getTransactionHashes().contains(islock.getTxId())) {
-                // Got an islock that didn't fit into the filtered block, so we must have received everything.
-                endFilteredBlock(currentFilteredBlock);
-                currentFilteredBlock = null;
-            } else {
-                //if this transaction is more than 24 hours old, don't process the islock
-                if (currentFilteredBlock.getBlockHeader().getTimeSeconds() + TimeUnit.DAYS.toSeconds(1) > System.currentTimeMillis()) {
-                    return;
-                }
-            }
-        }
-        context.instantSendManager.processInstantSendLock(this, islock);
-    }
+//    protected void processInstantSendLock(InstantSendLock islock) {
```

Comment:
I thought we need this 😆

Reply:
I better check this

Reply:
https://github.com/dashpay/dashj/pull/264/files/8782e0caffdc0f656a084a4685c1d99376e74e79#diff-c561608f57e133e4dd747b177af771172a9e33ed0ddd1979da7436e12cd092bfR260-R294

Message handling was moved out of peer into the specific manager class for each object.